FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -92,7 +101,7 @@
 /* Compile time error for some undefined configs, and provide default values
  * for others. */
 #ifndef democonfigMQTT_BROKER_ENDPOINT
-    #error "Please define democonfigMQTT_BROKER_ENDPOINT."
+    #error "Please define democonfigMQTT_BROKER_ENDPOINT in demo_config.h."
```

Comment:
If this is intended to work with AWS IoT, then supporting username/password along with metrics might be a good idea.

Reply:
I didn't want this demo to do too much since we have the mutual auth demo for that, though I did plan on using this with AWS IoT. How much work would that be?

Reply:
Just that I remember we wanted to have mutual auth demos automatically picking up metrics. Just requires copying and pasting a few macros and adding them to the connect info.